libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -626,8 +626,10 @@ void AP_GPS::update_instance(uint8_t instance)
             state[instance].hdop = GPS_UNKNOWN_DOP;
             state[instance].vdop = GPS_UNKNOWN_DOP;
             timing[instance].last_message_time_ms = tnow;
+            timing[instance].delta_time_ms = UINT16_MAX; // we are going to lose lock on what this is until after we have parsed 2 messages
```

Comment:
The first time we get a fix it's going to be an implausibly short. I will admit the comment is a bit confusing, but it is trying to hint that the first reading on this after getting a fix will be wrong.

Reply:
I understand your point now, but, besides being hard to understand, I think this is the wrong place to put such a comment. What about putting it two lines down where the clamp is? That's where the small implausible value will actually happen, right?

By the way, I think we should initialize the `delta_time_ms` to the same value instead of only when the driver is deleted.